project/Build.scala

```diff
@@ -1181,7 +1201,8 @@ object Build {
            * still compile even with only part of the files being present.
            */
           dependencyClasspath in doc ++= exportedProducts.value,
-
+    ))).zippedSettings(Seq("javalanglib", "javalib", "scalalib", "libraryAux"))(localProjects =>
```

Comment:
The first two `))` should be on the previous line and indented 2 more spaces, while the rest of the line should be indented 2 less spaces.